### DISCUSSION TITLE
Question about Mark down code

### DISCUSSION THREAD

**Discussion by keys-i** [community/community#55482] [Programming Help]
### Body

<a href="site link">
<img width="480" height="270" src="gif link" align=right>
</a>
- <img src="https://media.giphy.com/media/WUlplcMpOCEmTGBtBW/giphy.gif" width="30"> I enjoy doing
  - ‍📖 p1
  - 🔭 p2
  - 🔢 p3
  - 🖥️ p4
  - 🖼 p5
    - 🛠 p6
  - 🤖 p7
<p>  
doesnt show as expected but shows a para
  I was able to fix using html but how to fix without that
</p>
### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by jge162** [comment]
what is your question?
